checker/build.xml

```diff
@@ -203,6 +192,8 @@
             <path>
                 <fileset dir="${src}">
                     <include name="**/*.java"/>
+                    <!-- TODO: we actually want to build these sources with a Java 8 compiler. -->
```

Comment:
That would be a larger refactoring that shouldn't block availability of this feature.